pymc3/distributions/continuous.py

```diff
@@ -141,22 +286,85 @@ class Normal(Continuous):
     """
     def __init__(self, mu=0.0, tau=None, sd=None, *args, **kwargs):
         super(Normal, self).__init__(*args, **kwargs)
-        self.mean = self.median = self.mode = self.mu = mu
-        self.tau, self.sd = get_tau_sd(tau=tau, sd=sd)
+        self.mu, self.tau = self.parametrize(mu, tau, sd)
+        self.mean = self.median = self.mode = self.mu
         self.variance = 1. / self.tau
 
-    def logp(self, value):
+    def parametrize(self, mu=None, tau=None, sd=None):
+        try:
+            tau, sd = get_tau_sd(tau, sd)
+        except ValueError as err:
+            if check:
+                raise err
+        return mu, tau
+    
+    def pdf(self, x, point=None):
+        """
+        The probability density function (pdf) of the Normal distribution.
+        
+        Parameters
+        ----------
+            x : float
+                The value at which to evaluate the pdf.
+            point : dict
+                Parameter values to fix.                
+        """
+        mu, tau = draw_values([self.mu, self.tau], point=point)                                     
+        return np.sqrt(0.5 * tau / pi) * np.exp(-0.5 * tau * (x - mu) ** 2) 
```

Comment:
Any reason this is not theano? I think it's nice to have but I'm not sure where the PDF would be useful specifically. If it were to be used in modeling it would have to be Theano.